README.md

```diff
@@ -17,6 +17,15 @@ A web application for providing access to Digital Objects in SDR Storage.
   ./bin/test.sh
   ```
 
+### Note for osx users
+the current bundle requires eventmachine 1.0.7. This won't build on a modern mac
+without specifying where the ssl libraries are:
+
+```
+gem install eventmachine -v '1.0.7' -- --with-cppflags=-I/usr/local/opt/openssl/include
+```
+
```

Comment:
I'd prefer to try a bundle update to resolve this before adding this comment to the README, as it will inevitably become obsolete and need to be removed.  If you all insist on it being in the README, please separate this change to an isolated commit.